app/controllers/index.rb

```diff
@@ -0,0 +1,147 @@
+enable :sessions
+
+get '/' do
```

Comment:
your formatting makes this method extremely hard to read. Please make sure you're formatting your code correctly. Its extremely important.